### PR TITLE
Chassis: Subsystem and Command

### PR DESCRIPTION
**Pull request by MajorDarius**
PTAL

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -12,5 +12,9 @@
     ".project": true
   },
   "C_Cpp.default.configurationProvider": "vscode-wpilib",
-  "wpilib.online": true
+  "wpilib.online": true,
+  "C_Cpp.configurationWarnings": "Disabled",
+  "files.associations": {
+    "*.inc": "cpp"
+  }
```
Comment (duane701) [inline .vscode/settings.json]: I will take out this change.

File: src/main/cpp/OI.cpp [hunk 2]
```diff
@@ -6,9 +6,46 @@
 /*----------------------------------------------------------------------------*/
 
 #include "OI.h"
-
+#include "commands/TankDrive.h"
```
Comment (duane701) [inline src/main/cpp/OI.cpp]: Remove `#include "commands/TankDrive.h"`

File: src/main/cpp/subsystems/Chassis.cpp [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+Chassis::Chassis() : Subsystem(kSubsystemName),
+  //defaultCommand(nullptr),
+  right1Wheel{RobotMap::kIDRight1Wheel, rev::CANSparkMaxLowLevel::MotorType::kBrushless},
+  right2Wheel{RobotMap::kIDRight2Wheel, rev::CANSparkMaxLowLevel::MotorType::kBrushless},
+  left1Wheel{RobotMap::kIDLeft1Wheel, rev::CANSparkMaxLowLevel::MotorType::kBrushless},
+  left2Wheel{RobotMap::kIDLeft2Wheel, rev::CANSparkMaxLowLevel::MotorType::kBrushless}
+  {
+    left2Wheel.Follow(left1Wheel);
+	  right2Wheel.Follow(right1Wheel);
```
Comment (duane701) [inline src/main/cpp/subsystems/Chassis.cpp]: Remove tabs. Let's watch for tabs.

File: src/main/cpp/subsystems/Chassis.cpp [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+void Chassis::InitDefaultCommand() {
+  // Set the default command for a subsystem here.
+  SetDefaultCommand(new TankDrive);
+}
+
+void Chassis::SetTankDrive(double left, double right) {
+  left1Wheel.Set(left);
+  right1Wheel.Set(right);
+  left2Wheel.Set(left);
+  right2Wheel.Set(right);
```
Comment (duane701) [inline src/main/cpp/subsystems/Chassis.cpp]: Don't call `Set` for `left2Wheel` and `right2Wheel`. These controllers are already followers.

File: src/main/include/OI.h [hunk 5]
```diff
@@ -1,13 +1,35 @@
-/*----------------------------------------------------------------------------*/
-/* Copyright (c) 2017-2018 FIRST. All Rights Reserved.                        */
-/* Open Source Software - may be modified and shared by FRC teams. The code   */
-/* must be accompanied by the FIRST BSD license file in the root directory of */
-/* the project.                                                               */
-/*----------------------------------------------------------------------------*/
+#ifndef OI_H
+#define OI_H
```
Comment (duane701) [inline src/main/include/OI.h]: Don't use guards. Use `#pragma once`

File: src/main/include/Robot.h [hunk 6]
```diff
@@ -1,3 +1,6 @@
+#ifndef _ROBOT_H
+#define _ROBOT_H
```
Comment (duane701) [inline src/main/include/Robot.h]: Don't use guards.

File: .vscode/settings.json (hunk 1 above)
Comment (duane701) [reply .vscode/settings.json]: I will remove this change on the rebase.

**Comment by AnekSahota** [comment]
REPLACED BY #15

**Pull request closed without merging**